R/Api.R

```diff
@@ -322,7 +322,7 @@ get_user_agent <- function() {
 serialize_parameters <- function(parameters) {
   serialized_params <- list()
   for (param in names(parameters)) {
```

Comment:
We can try to use a deprecation warning here too.  E.g. put `param == "genre" ||` back in the `if` statement, and add this `if` clause in the same `for` loop.
```
    if (param == "genre") {
      warning("The genre parameter is deprecated and will be removed in a future release.")
    }
```